packages/forms-web-app/__tests__/unit/controllers/examination/check-your-answers/utils/summary-list-item/show-deadline-item-change-url.test.js

```diff
@@ -0,0 +1,52 @@
+const {
+	showDeadlineItemChangeUrl
+} = require('../../../../../../../src/controllers/examination/check-submission-item/utils/summary-list-item/show-deadline-item-change-url');
+
+describe('controllers/examination/check-submission-item/utils/summary-list-item/show-deadline-item-change-url', () => {
+	describe('#showDeadlineItemChangeUrl', () => {
+		describe('When getting the boolean value to show or hide the change deadline item url for the summary list item for the check your answers page', () => {
+			describe('and submission items has the same number of items as deadline items', () => {
+				const session = {
```

Comment:
these should be mocks to the two getters. 

Otherwise large shapes of session data is being passed around the tests. If that structure changes this test would have to change. Not just the getter.